eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -132,6 +132,15 @@ void getsAlreadyOptimizedResultsFromCache(@TempDir final Path temp) throws Excep
                 .resolve(hash)
                 .resolve("foo/x/main.xmir")
         );
+        Files.setLastModifiedTime(
```

Comment:
@Yanich96 Can we use `save()` function instead?

Reply:
@volodya-lombrozo Is it necessary to create a new function here just for one use?

Reply:
@volodya-lombrozo I will create issue